pkg/cluster/resources.go

```diff
@@ -119,13 +121,74 @@ func (c *Cluster) createStatefulSet() (*v1beta1.StatefulSet, error) {
 	return statefulSet, nil
 }
 
+func getPodNumb(podName string) (int32, error) {
```

Comment:
I'd also name it getPodIndex, as getPodNumb reads like you want to make the Pod numb.